scripts/build-tools.js

```diff
@@ -22,10 +22,10 @@ const buildTools = async (automatedToolsPath, manualToolsPath, toolsPath, tagsPa
 
 /* istanbul ignore next */
 if (require.main === module) {
-  const automatedToolsPath = resolve(__dirname, '../config', 'tools-automated.json');
-  const manualToolsPath = resolve(__dirname, '../config', 'tools-manual.json');
-  const toolsPath = resolve(__dirname, '../config', 'tools.json');
-  const tagsPath = resolve(__dirname, '../config', 'all-tags.json');
```

Comment:
Why has `resolve` worked previously? I saw tests or scripts aren't failing even with `resolve`

Reply:
I thought the same we dont need this change, the workflow was initially failing on Windows, so I researched it and found that it might be due to path errors. I came across a solution suggesting the use of path.join, which should work across all OS. However, it's now failing on Ubuntu as well, and I think the issue might be with the fs module, which is causing the workflow to fail.

Reply:
Should i close this PR ?

Reply:
Don't close it but try to identify the root cause of that issue and push the changes